rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/WorkerSourceTask.java

```diff
@@ -167,8 +167,8 @@ public void run() {
                     try {
                         toSendRecord = poll();
                         if (null != toSendRecord && toSendRecord.size() > 0) {
-                            connectStatsManager.incSourceRecordPollTotalNums();
-                            connectStatsManager.incSourceRecordPollNums(taskConfig.getString(RuntimeConfigDefine.TASK_ID));
+                            connectStatsManager.incSourceRecordPollTotalNums(toSendRecord.size());
```

Comment:
Shouldn't a new statistical dimension be defined? At present, SourceRecordPollTotalNums represents the total number of source task calls, and SourceRecordPollNums will record the number of data for each task